Bug: dmp:tag executed from a POM fails on images without `<build>` (0.46)

I have a build which runs 2 images, and creates 1.

The key word is **creates** because it's via `docker commit`.

After creating it, I also want to tag the image, for which I would like to use DMP rather than another invocation of `docker`.

```xml
                                <image>
                                    <name>${nx.gitlabImagesRepo}:granit-latest</name>
                                    <build>
                                        <from>${nx.gitlabImagesRepo}:granit-latest</from>
                                        <skip>true</skip><!-- Skip building, not pushing. See skipTag and skipPush. -->
                                        <tags><tag>granit-latest</tag></tags>
                                    </build>
                                </image>
```

But, DMP fails on the 2 `<images>`s not having a `<build>`:

```
[INFO] --- docker:0.46.0:tag (granit-image-tag-dmp) @ config-hub-granit-schemas ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  34.837 s
[INFO] Finished at: 2025-05-21T01:28:49+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.46.0:tag (granit-image-tag-dmp) on project config-hub-granit-schemas: Execution granit-image-tag-dmp of goal io.fabric8:docker-maven-plugin:0.46.0:tag failed: Cannot invoke "io.fabric8.maven.docker.config.BuildImageConfiguration.skipTag()" because "buildConfig" is null -> [Help 1]
```

When I remove the 2 `<run>`-only images,  tagging works fine.

Workaround:

Add this to the `<run>`-only images:

```xml
<build><skip>true</skip><skipTag>true</skipTag><skipPush>true</skipPush></build>
```

Solution:

Make `dmp:tag` skip images with no `<build>`.

--- a/src/main/java/io/fabric8/maven/docker/TagMojo.java
+++ b/src/main/java/io/fabric8/maven/docker/TagMojo.java
@@ -35,7 +35,8 @@ public class TagMojo extends AbstractDockerMojo {
         List<ImageConfiguration> imageConfigs = getResolvedImages();
         for (ImageConfiguration imageConfig : imageConfigs) {
             BuildImageConfiguration buildConfig = imageConfig.getBuildConfiguration();
-            if (buildConfig.skipTag() || buildConfig.isBuildX()) {
+
+            if (buildConfig == null || buildConfig.skipTag() || buildConfig.isBuildX()) {
                 // Tag happens at the building stage.
                 continue;
             }

